### PR TITLE
DOCSP 16448 compatibility updates

### PR DESCRIPTION
**Pull request by rustagir**
### Issue JIRA link:
https://jira.mongodb.org/browse/DOCSP-16448

### Docs staging link (requires sign-in on MongoDB Corp SSO):
https://docs-mongodborg-staging.corp.mongodb.com/drivers/docsworker-xlarge/DOCSP-16448-compatibility-updates/

### CODE REVIEW

**Review by terakilobyte — APPROVED** [review]
LGTM. We'll hold off on merging this until closer to .live

**Comment by rustagir** [comment]
Removed changes for these drivers:

- Motor
- Swift
- C++

File: source/includes/language-compatibility-table-php.rst [hunk 1]
```diff
@@ -15,7 +15,7 @@
      - PHP 5.6
      - PHP 5.5
 
-   * - ext 1.9 + lib 1.8
```
Comment (terakilobyte) [inline source/includes/language-compatibility-table-php.rst:18]: Just confirming that there is no `ext 1.9` and `lib 1.8` ?

**Review by terakilobyte — APPROVED** [review]
LGTM

File: source/includes/language-compatibility-table-php.rst (hunk 1 above)
Comment (jmikola) [reply source/includes/language-compatibility-table-php.rst:18]: I don't understand the question. This PR is leaving "ext 1.9 + lib 1.8" as-is and only introducing a new row for "ext 1.10 + lib 1.9".

File: source/includes/php-mongodb-compatibility-table.rst [hunk 2]
```diff
@@ -13,7 +14,19 @@
      - MongoDB 3.0
      - MongoDB 2.6
 
+   * - ext 1.10 + lib 1.9
+     - |checkmark|
```
Comment (jmikola) [inline source/includes/php-mongodb-compatibility-table.rst:18]: IIRC, previous versions of these server compatibility tables had footnotes when driver releases were not fully compatible. In this case, PHPC 1.10 and PHPLIB 1.9 will be missing support for snapshot reads and load balancers.

AFAIK, load balancers are not specifically tied to server 5.0, but should we have a footnote about lacking support for snapshot reads? The plan for any drivers missing features is to release what we have on Tuesday, July 13 and follow up with a subsequent minor release ASAP with the remaining work.

Also: I'm only aware of the case for PHP, but it's possible another driver or two is in a similar boat (with possibly different features).

cc: @alcaeus

File: source/includes/language-compatibility-table-php.rst (hunk 1 above)
Comment (terakilobyte) [reply source/includes/language-compatibility-table-php.rst:18]: I can resolve this. Initially the versions were updated and no new row added, making it appear that these versions were skipped

File: source/includes/php-mongodb-compatibility-table.rst (hunk 2 above)
Comment (rustagir) [reply source/includes/php-mongodb-compatibility-table.rst]: Hi Jeremy,

I added a footnote to the PHP compatibility table, see here: 

https://docs-mongodbcom-staging.corp.mongodb.com/drivers/docsworker-xlarge/DOCSP-16448-compatibility-updates/php/#mongodb-compatibility

Does that impart the correct/enough information?

Comment (jmikola) [reply source/includes/php-mongodb-compatibility-table.rst]: The corresponding drivers ticket is [DRIVERS-815](ttps://jira.mongodb.org/browse/DRIVERS-815), where this is referred to as "Snapshot reads on secondaries"; however, I'm not in a position to say whether that is the correct customer-facing name. The 5.0 server release notes refer to [Snapshots](https://docs.mongodb.com/v5.0/release-notes/5.0/#snapshots), which I presume is the same feature.

I would suggest one of the following:

 * Refer to "snapshot reads on secondaries"
 * Refer to "Snapshots" and link to the release notes for context

@terakilobyte: Can you weigh in here? @techbelle may also be in a better position to make the call on how best to refer to this.

Comment (rustagir) [reply source/includes/php-mongodb-compatibility-table.rst]: Staging links:

C: https://docs-mongodbcom-staging.corp.mongodb.com/drivers/docsworker-xlarge/DOCSP-16448-compatibility-updates/c/#footnote-c-1.18-driver-support

Go: https://docs-mongodbcom-staging.corp.mongodb.com/drivers/docsworker-xlarge/DOCSP-16448-compatibility-updates/go/#footnote-go-1.6-driver-support

PHP: https://docs-mongodbcom-staging.corp.mongodb.com/drivers/docsworker-xlarge/DOCSP-16448-compatibility-updates/php/#footnote-PHPC1.10-PHPLIB1.9-driver-support

**Pull request merged**